src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -1,22 +1,88 @@
 package codeu.controller;
 
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
```

Comment:
Nit: these two imports (User and UserStore) are duplicated below. You can remove them from here, and then they'll all be in nice alphabetical order.